src/main/kotlin/org/rust/ide/refactoring/changeSignature/RsChangeSignatureDialog.kt

```diff
@@ -399,3 +429,17 @@ private class VisibilityComboBox(project: Project, initialVis: RsVis?, onChange:
 
 private fun createVisibilityHints(initialVis: RsVis?): Array<String> =
     setOf(initialVis?.text.orEmpty(), "", "pub", "pub(crate)", "pub(super)").toTypedArray()
+
+private fun RsElement.createImportContext(): RsMod {
+    val factory = RsPsiFactory(project)
+    val module = factory.createModItem(TMP_MOD_NAME, "")
+    val sourceContext = containingFile as RsFile
+
+    val qualifiedPath = containingMod.qualifiedNameRelativeTo(sourceContext)
```

Comment:
Shouldn't we use absolute path here (`qualifiedNameInCrate`)?

Reply:
I checked the (auto)imported imports and they are relative to the containing file (probably because of `setContext`). So that's why I used the same import style (I think that absolute imports didn't work). However, now that I think of it, it might have been caused by the usage of edition 2015. I'll test it in both editions to find a solution that works for both.

Reply:
Thank you, `qualifiedNameInCrate` works in both editions.